src/llm_vm/completion/optimize.py

```diff
@@ -4,63 +4,68 @@
 import time
 import os
 import sys
-import signal 
+import signal
 import json
 import tempfile
 import abc
 import requests
 import hashlib
 import pickle
 import data_synthesis
+import tiktoken
+
+job_id = None  # we want to be able to cancel a fine_tune if you kill the program
 
-job_id = None # we want to be able to cancel a fine_tune if you kill the program
 
 def exit_handler(signum, frame):
-    
-    if (None != job_id):
+    if None != job_id:
         print("cancelling fine-tune if applicable")
         openai.FineTune.cancel(id=job_id)
 
     print("user interrupt, exiting")
     sys.exit()
 
+
 signal.signal(signal.SIGINT, exit_handler)
 
 
 def generate_hash(input_string):
     sha256_hash = hashlib.sha256()
-    sha256_hash.update(str(input_string).encode('utf-8'))
+    sha256_hash.update(str(input_string).encode("utf-8"))
```

Comment:
why are the quotes changed? did you run some automatic formater on all the files?